src/index.html

```diff
@@ -15,8 +15,94 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#moyo"
+        class="img"
+      >
+        <img
+          width="100%"
+          height="100%"
+          alt="moyo"
+          src="/src/images/logo.png"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```